pages/common/enca.md

```diff
@@ -0,0 +1,23 @@
+# enca
+
+> Detect and convert encoding of text files
+
+- detect file encoding according to your system's locale
+
+`enca {{file}}`
+
+- detect file encoding; -L option tells enca the current language; `language` is zh_CN, en_US etc.
+
+`enca -L {{language}} {{file}}`
+
+- convert file to specified encoding
+
+`enca -L {{language}} -x {{to_encoding}} {{file}}`
+
+- convert multiple files to specified encoding
+
+`enca -L {{language}} -x {{to_encoding}} *`
+
+- convert file to specified encoding and save to new file
+
+`enca -L {{language}} -x {{to_encoding}} {{original_file}} {{new_file}}`
```

Comment:
This one is actually incorrect. I got `enca: Cannot open file `new_file' in mode r+b: No such file or directory` when there is only `original_file` in the current folder. Also the syntax described here is ambiguous with the last two examples.